### PR TITLE
ENH add single data fecther with OSF

### PR DESCRIPTION
**Pull request by tomMoral**
A single data fetcher, hopefully usable easily for other challenges.

### CODE REVIEW

File: download_data.py [hunk 1]
```diff
@@ -1,34 +1,143 @@
...
+        assert data_exists(private), (
+            f"{LOCAL_DATA} already exists but the checksum does not match. "
+            f"Please try to remove the folder {LOCAL_DATA} and rerun this "
+            "command to get the proper data."
+        )
+
 
 if __name__ == "__main__":
-    download_data()
+    import argparse
```
Comment (maikia) [inline download_data.py]: is there any reason why you import here and not on the top?

**Review by agramfort — APPROVED** [review]
LGTM !

@maikia I let you merge if you're happy

thx @tomMoral

File: download_data.py [hunk 2]
```diff
@@ -1,34 +1,143 @@
...
+        )
+
+        # Find the folder in the OSF project
+        print("Checking the data URL...", end='', flush=True)
+        challenge_folder = get_one_element(store.folders, CHALLENGE_NAME)
+
+        # Find the file to download from the OSF project
+        archive_name = data_config['archive_name']
+        osf_file = get_one_element(challenge_folder.files, archive_name)
+        print('Ok.')
```
Comment (maikia) [inline download_data.py:110]: if you want to print something maybe better something more informative? same in other locations

**Review by maikia — APPROVED** [review]
LGTM

**Comment by maikia** [comment]
thanks @tomMoral 
once the CI passes we can merge

File: download_data.py (hunk 2 above)
Comment (tomMoral) [reply download_data.py:110]: This is the end of the print statement on line 98 :smile:  . so it gives something like:

```
Checking the data...Ok.
```

**Review by deepcharles — APPROVED** [review]
Looks good. I'll wait for the data upload to finish to test it on private data.

**Pull request merged**